dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -851,8 +855,9 @@ protected Response saveContent(Contentlet contentlet, InitDataObject init) throw
 									hu.setQuery("from "+Category.class.getCanonicalName()+" WHERE category_velocity_var_name=?");
 									hu.setParam(cat);
 									category=(Category)hu.load();
-									if(category!=null && InodeUtils.isSet(category.getCategoryId())) {
-										cats.add(category);
+                                    if(category!=null && InodeUtils.isSet(category.getCategoryId()) 
+                                            && APILocator.getPermissionAPI().doesUserHavePermission(category, PermissionAPI.PERMISSION_USE, init.getUser(), allowFrontEndSaving)) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 179 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)